### PR TITLE
Removes an unused species variable

### PR DESCRIPTION
**Pull request by ChungusGamer666**
## About The Pull Request

Title.

## Why It's Good For The Game

Why waste memory for a variable that is never used?

## Changelog

no

### CODE REVIEW

**Comment by necromanceranne** [comment]
I think only material golems used it, and they're long gone.

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -314,12 +314,8 @@
...
 		var/sound_to_play = 'sound/weapons/thudswoosh.ogg'
-		if(ishuman(src))
-			var/mob/living/carbon/human/H = src
-			if(H.dna.species.grab_sound)
-				sound_to_play = H.dna.species.grab_sound
-			if(HAS_TRAIT(H, TRAIT_STRONG_GRABBER))
-				sound_to_play = null
+		if(ismob(src) && HAS_TRAIT(src, TRAIT_STRONG_GRABBER))
+			sound_to_play = null
 		playsound(src.loc, sound_to_play, 50, TRUE, -1)
```
Comment (lordpidey) [inline code/modules/mob/living/living.dm:319]:
```
if(!ismob(src) || !HAS_TRAIT(src ,TRAIT_STRONG_GRABBER))
    playsound(src.loc, sound_to_play, 50, TRUE, -1)
```

**Review by lordpidey — CHANGES_REQUESTED** [review]
Code can be made a bit cleaner, rather than having an if statement to null the sound file to play, instead have the if statement reversed, and have it play the sound.

File: code/modules/mob/living/living.dm (hunk 1 above)
Comment (ChungusGamer666) [reply code/modules/mob/living/living.dm:319]: truth
facts even
will fix when i can

**Comment by san7890** [comment]
instead of doing this you should instead make it so that every time you're grabbed by a felinid a mew sound plays. or like fluttering from a moth wings. maybe zapping sounds for an ethereal. a variable that stays the same as its compile-time value will take up absolutely zero memory so the only reason to remove this is cleanliness.

idk i think it's an okayish framework and if you really detest the variable we can just switch it from an element to a variable that acts upon a new signal or something. doesn't really need to be implemented fully, but i'd rather we do the whole "improve don't remove:tm:" thing that /tg/ is famous for

**Comment by ChungusGamer666** [comment]
> idk i think it's an okayish framework and if you really detest the variable we can just switch it from an element to a variable that acts upon a new signal or something. doesn't really need to be implemented fully, but i'd rather we do the whole "improve don't remove™️" thing that /tg/ is famous for

yeah its cleanliness tbh but like this variable is currently unused and i dont think anyone is gonna use it any soon idk
i do think actually using it would be cute but i wasnt planning on it

**Comment by ChungusGamer666** [comment]
imma close this pr, maybe in the future i can actually use this variable for cute species grab sound shit

**Pull request closed without merging**